src/eventSchemas/explorer/schemasNode.ts

```diff
@@ -27,23 +28,24 @@ export class SchemasNode extends AWSTreeErrorHandlerNode {
         this.contextValue = 'awsSchemasNode'
     }
 
-    public async getChildren(): Promise<(RegistryItemNode | ErrorNode | PlaceholderNode)[]> {
-        await this.handleErrorProneOperation(
-            async () => this.updateChildren(),
-            localize('AWS.explorerNode.schemas.error', 'Error loading Schemas resources')
-        )
-
-        if (this.errorNode) {
-            return [this.errorNode]
-        }
-
-        if (this.registryNodes.size > 0) {
-            return [...this.registryNodes.values()].sort((nodeA, nodeB) =>
+    public async getChildren(): Promise<AWSTreeNodeBase[]> {
+        return await makeChildrenNodes({
+            getChildNodes: async () => {
+                await this.updateChildren()
+
+                return [...this.registryNodes.values()]
+            },
+            getErrorNode: async (error: Error) =>
+                new ErrorNode(
+                    this,
+                    error,
+                    localize('AWS.explorerNode.schemas.error', 'Error loading Schemas resources')
+                ),
+            getNoChildrenPlaceholderNode: async () =>
+                new PlaceholderNode(this, localize('AWS.explorerNode.schemas.noRegistry', '[No Schema Registries]')),
```

Comment:
Nit: this is capitalized, but the registryItemNode's message isn't. Which one is correct?

Reply:
Good catch . The lowercase text is not consistent and looks less polished. Will revise.